guides/theming.md

```diff
@@ -1,311 +1,359 @@
+custom theme with Sass, or by importing a pre-built theme CSS file.
 
+### Custom themes with Sass
+
+A **theme file** is a Sass file that calls Angular Material Sass mixins to output color and
+typography CSS styles.
+
+#### The `core` mixin
+
+Angular Material defines a mixin named `core` that includes prerequisite styles for common
+features used by multiple components, such as ripples. The `core` mixin must be included exactly
+once for your application, even if you define multiple themes.
 
-A typical theme file will look something like this:
 ```scss
-@import '~@angular/material/theming';
-// Plus imports for other components in your app.
-
-// Include the common styles for Angular Material. We include this here so that you only
-// have to load a single css file for Angular Material in your app.
-// Be sure that you only ever include this mixin once!
-@include mat-core();
-
-// Define the palettes for your theme using the Material Design palettes available in palette.scss
-// (imported above). For each palette, you can optionally specify a default, lighter, and darker
-// hue. Available color palettes: https://material.io/design/color/
-$candy-app-primary: mat-palette($mat-indigo);
-$candy-app-accent:  mat-palette($mat-pink, A200, A100, A400);
-
-// The warn palette is optional (defaults to red).
-$candy-app-warn:    mat-palette($mat-red);
-
-// Create the theme object. A theme consists of configurations for individual
-// theming systems such as `color` or `typography`.
-$candy-app-theme: mat-light-theme((
-  color: (
-    primary: $candy-app-primary,
-    accent: $candy-app-accent,
-    warn: $candy-app-warn,
-  ),
-  typography: mat-typography-config(),
-  density: 0, // Defaults to 0 if omitted, but shown for completeness.
-));
+@use '~@angular/material' as mat;
 
-// Include theme styles for core and each component used in your app.
-// Alternatively, you can import and @include the theme mixins for each component
-// that you are using.
-@include angular-material-theme($candy-app-theme);
+@include mat.core();
 ```
 
-You only need this single Sass file; you do not need to use Sass to style the rest of your app.
+#### Defining a theme
 
-If you are using the Angular CLI, support for compiling Sass to css is built-in; you only have to
-add a new entry to the `"styles"` list in `angular.json` pointing to the theme
-file (e.g., `unicorn-app-theme.scss`).
+Angular Material represents a theme as a Sass map that contains your color and typography
+choices. For more about typography customization, see [Angular Material Typograpy][mat-typography].
 
-If you're not using the Angular CLI, you can use any existing Sass tooling to build the file (such
-as gulp-sass or grunt-sass). The simplest approach is to use the `sass` CLI; you simply run:
-```
-sass src/unicorn-app-theme.scss dist/unicorn-app-theme.css
+Constructing the theme first requires defining your primary and accent palletes, with an optional
```

Comment:
Should "primary", "accent" and "warn" be wrapped in backticks so it's more obvious that they refer to code?

Reply:
other places in this doc you quote them: "accent"

Reply:
I don't think code ticks are warranted here because I'm referring to concept of the palettes. I also changed my quotes where I define the terms to bold, which is more consistent with other term definitions.